geometry_msgs/PoseStamped targPoint
geometry_msgs/PoseStamped pose
geometry_msgs/TwistStamped twist
sensor_msgs/Imu imuData
bool isUpdateTarget
---
mavros_msgs/PositionTarget setPointRaw
bool success